Production/test/condorSub/dict_Summer20UL16APV_tt.py

```diff
@@ -0,0 +1,6 @@
+flist = {
+    "scenario": "Summer20UL16APV",
+    "samples": [
+        ['Summer20UL16APV.TTToHadronic_TuneCP5_13TeV-powheg-pythia8'],
```

Comment:
Why did TTToHadronic get put in this file when the other TTJets samples got put into the ttbar file?

Reply:
I think this was just a disconnect between when I made "example" changes to include a new sample and when @semrat started adding more samples in. We will of course consolidate and clean this up.